docs/libindy/android.md

```diff
@@ -0,0 +1,46 @@
+# Setup Libindy for Android
+
+> NOTE: If these steps do not work, or you want to do a manual install, refer to [here](https://github.com/hyperledger/indy-sdk#android)
+
+## Android
+
+1. Go to the [release page](https://github.com/animo/libindy-binaries) and download the Android binaries
+   - These assets are acquired from [here](https://repo.sovrin.org/android/libindy/stable/1.16.0/)
```

Comment:
```suggestion
   - These assets are acquired from the [Sovrin repo](https://repo.sovrin.org/android/libindy/stable/1.16.0/)
```

Reply:
I.p.v. `here` the gebruiken voor links probeer ik altijd te omschrijven waar de link heen gaat